text/SUMMARY.md

```diff
@@ -2,37 +2,38 @@
 
 [Introduction](./introduction.md)
 - [Pallets](./pallets-intro.md)
-	- [Printing to the Node Log](./runtime-printing.md)
-	- [Emitting Events](./events.md)
-	- [Storage Maps](./storage-maps.md)
-	- [Cache Locally > Storage Calls](./cache.md)
-	- [Using Vectors as Sets](./vec-set.md)
-	- [Using Maps as Sets](./map-set.md)
-	- [Subgroup Removal by Subkey: Double Maps](./double.md)
-	- [Storing custom structs](./structs.md)
-	- [Ringbuffer Queue](./ringbuffer.md)
-	- [Basic Token](./basic-token.md)
-	- [Configurable Constants](./constants.md)
-	- [Simple Crowdfund](./crowdfund.md)
-	- [Instantiable Pallets](./instantiable.md)
-	- [Weights for Resource Accounting](./weights.md)
-	- [Charity and Imbalances](./charity.md)
-	- [Fixed Point Arithmetic](./fixed-point.md)
-	- [Off-chain Workers](./off-chain-workers/index.md)
-		- [Transactions](./off-chain-workers/transactions.md)
-		- [HTTP Fetching & JSON Parsing](./off-chain-workers/http-json.md)
-		- [Local Storage](./off-chain-workers/storage.md)
-	- [Currency Types](./currency.md)
-	- [Currency and Imbalances](./currency-imbalances.md)
-	- [Generating Randomness](./randomness.md)
-	- [Tightly- and Loosely-Coupled Pallets](./pallet-coupling.md)
+  - [Printing to the Node Log](./runtime-printing.md)
+  - [Emitting Events](./events.md)
+  - [Storage Maps](./storage-maps.md)
+  - [Cache Locally > Storage Calls](./cache.md)
+  - [Using Vectors as Sets](./vec-set.md)
+  - [Using Maps as Sets](./map-set.md)
+  - [Subgroup Removal by Subkey: Double Maps](./double.md)
+  - [Storing custom structs](./structs.md)
+  - [Ringbuffer Queue](./ringbuffer.md)
+  - [Basic Token](./basic-token.md)
+  - [Configurable Constants](./constants.md)
+  - [Simple Crowdfund](./crowdfund.md)
+  - [Instantiable Pallets](./instantiable.md)
+  - [Weights for Resource Accounting](./weights.md)
+  - [Charity and Imbalances](./charity.md)
+  - [Fixed Point Arithmetic](./fixed-point.md)
+  - [Off-chain Workers](./off-chain-workers/index.md)
+    - [Transactions](./off-chain-workers/transactions.md)
```

Comment:
Looks like some blankspace changes. Are these intentional?

Reply:
This is due to the `.editorconfig` we are using now. Only `.rs` and `.toml` files are saved with tab. Files with other extension will have tabs converted to two spaces.